_sass/modules/_m-card--indexes.scss

```diff
@@ -108,7 +111,10 @@
       border-right: 0;
     }
     &:nth-child(2n + 1) {
-      border-right: 1px solid $border-3;
+      // border-right: 1px solid $border-3;
     }
   }
+  @media screen and (min-width: $screen-l) {
+    height: rem(600px);
+  }
```

Comment:
I think you are not taking into account paddings height and that is why it seems too tall.